package.json

```diff
@@ -3,19 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
```

Comment:
Testing-library should probably be under dev dependencies :)

Reply:
@wooneh weirdly this the CRA way I think we'll leave them for now ... see https://github.com/facebook/create-react-app/issues/8210